spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/support/converter/SimplePubSubMessageConverter.java

```diff
@@ -0,0 +1,90 @@
+/*
+ *  Copyright 2018 original author or authors.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.springframework.cloud.gcp.pubsub.support.converter;
+
+import java.nio.charset.Charset;
+import java.util.Map;
+
+import com.google.protobuf.ByteString;
+import com.google.pubsub.v1.PubsubMessage;
+
+/**
+ * A simple {@link SimplePubSubMessageConverter} that that directly maps payloads of type
+ * {@code byte[]}, {@code ByteString}, and {@code String} to Pub/Sub messages.
+ *
+ * @author Mike Eltsufin
+ */
+public class SimplePubSubMessageConverter implements PubSubMessageConverter {
+
+	private Charset charset = Charset.defaultCharset();
+
+	/**
+	 * Set charset for converting strings.
+	 */
+	public void setCharset(Charset charset) {
```

Comment:
Can't we have a `charset` as a ctor arg?
One ctor is without args and delegates to that one with the `charset`.
Much efficient during runtime since that `charset` propety is going to be `final` and it is much convenient to declare instances of this class.

Thanks